src/components/Typography/Typography.tsx

```diff
@@ -26,30 +23,32 @@ type TypographyType = React.FC<TypographyProps> & {
   Block: React.FC<TypographyProps>;
 };
 const Typography: TypographyType = ({
-  align = 'inherit',
   color = 'primary',
   className: passedClassName = '',
-  size = 'md',
+  size = 'medium',
```

Comment:
Curious... Why is this defaulted `medium` instead of `md`? The style class is defined as `.size--md` and the size type doesn't have a medium option.

Reply:
Oh, I didn't mean to do that! Thank you for catching it!